nupic/research/frameworks/pytorch/models/common.py

```diff
@@ -0,0 +1,41 @@
+#  Numenta Platform for Intelligent Computing (NuPIC)
+#  Copyright (C) 2019, Numenta, Inc.  Unless you have an agreement
+#  with Numenta, Inc., for a separate license for this software code, the
+#  following terms and conditions apply:
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU Affero Public License version 3 as
+#  published by the Free Software Foundation.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#  See the GNU Affero Public License for more details.
+#
+#  You should have received a copy of the GNU Affero Public License
+#  along with this program.  If not, see http://www.gnu.org/licenses.
+#
+#  http://numenta.org/licenses/
+#
+
+import numpy as np
+import torch.nn as nn
+
+from nupic.torch.modules import Flatten
```

Comment:
pytorch 1.6 introduced [torch.nn.Flatten](https://github.com/pytorch/pytorch/blob/master/torch/nn/modules/flatten.py). We should start using pytorch version instead of `nupic.torch`

Reply:
Good point, changed.